src/DynamoCore/Models/DynamoModelEvents.cs

```diff
@@ -203,6 +203,17 @@ public virtual void OnEvaluationCompleted(object sender, EvaluationCompletedEven
                 EvaluationCompleted(sender, e);
         }
 
+        /// <summary>
+        /// An event triggered when all tasks in scheduler are completed.
+        /// </summary>
+        public event Action<HomeWorkspaceModel> RefreshCompleted;
```

Comment:
Hi @pboyer ,
I'm not sure if it is exactly what you want me to do... please, take a look.